javascript/javascript3/week2/style.css

```diff
@@ -0,0 +1,58 @@
+* {
```

Comment:
Use Prettier also for css files :)

Reply:
Nice that the webpage is responsive.

Reply:
Hi Svetlana,
Could you please point out what is not properly formatted in the CSS file?
I have checked all my configurations in VSCode, and as far as I could see, the CSS file is being formatted.

Reply:
fetch .then .catch were taught in class at first.
Subsequently, we learned the try/catch method by replacing the .then with a variable to fetch the data.
We were told that this method is cleaner and shorter.
That's why I used try/catch instead of .then .catch